website/src/parsers/mdx/mdx2recma.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'mdx2/package.json';
+
+const ID = 'mdx2-recma';
```

Comment:
I'd either use the names `mdast` / `hast` / `estree` or `remark` / `rehype` / `recma`, but not a mix of those.

Reply:
yeah, same, should be `estree` probably

Reply:
Good point, updated it to be named `esast` (https://github.com/syntax-tree/esast)